package.json

```diff
@@ -68,6 +68,7 @@
     "webpack-dev-server": "^1.12.0"
   },
   "dependencies": {
+    "classnames": "^2.2.3",
```

Comment:
this belongs in `devDependencies` - `npm install --save-dev classnames`
also we should probably discuss new packages before adding them just to avoid package bloat... but I also just chucked `react-redux` into our dependencies too so.